src/prefect/environments/execution/remote/environment.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any
+
+import cloudpickle
+
+from prefect.environments.execution import Environment
+from prefect.environments.storage import Docker
+from prefect.utilities import logging
+from prefect.utilities.configuration import set_temporary_config
+
+
+class RemoteEnvironment(Environment):
+    """
+    RemoteEnvironment is an environment which takes in information around an executor
+    and runs the flow in place using that executor.
+
+    Example using a RemoteEnvironment w/ an existing Dask cluster:
+    ```
+    RemoteEnvironment(
```

Comment:
Could you maybe store this as `env` and actually attach it to a flow?  Just for added documentation about how to specify environments.